dispatch/dispatch.h

```diff
@@ -32,6 +32,10 @@
 #include <stdarg.h>
 #include <unistd.h>
 #include <fcntl.h>
+#ifndef __APPLE__
+#include <stdio.h>
```

Comment:
why do you need that in a public header? we kind of try to restrict what we export to a bare minimum, <sys/types.h> I understand, but stdio seems to point toward something else being wrong.

I would also add <sys/types.h> for everyone not only !__APPLE__ it's fine